pkg/virt-handler/vm.go

```diff
@@ -111,6 +111,12 @@ type netstat interface {
 	CachePodInterfaceVolatileData(vmi *v1.VirtualMachineInstance, ifaceName string, data *netcache.PodIfaceCacheData)
 }
 
+type genericMetricsManager interface {
```

Comment:
nit: Rename to `DownwardMetricsManager`?

Reply:
Does it have to be public?

Reply:
No, of course it can stay private.

Reply:
done